ocs_ci/ocs/disruptive_operations.py

```diff
@@ -0,0 +1,62 @@
+import logging
```

Comment:
I've just found that we have:  tests/disruption_helpers.py - what do you think to consolidate those two modules?

Reply:
i can move my code into this file for now, but in my opinion we should discuss having helpers files within tests/ folder

Reply:
I agree with not have them in tests folder.  For now you can have them as you have, but open issue and in some next PR move maybe helper functions from tests/disruption_helpers.py  to your new module?